aggregate/coretests-site/pom.xml

```diff
@@ -73,6 +73,58 @@
     </plugins>
   </build>
 
+  <profiles>
+  	<profile>
+		<id>deploy-to-jboss.org</id>
+		<build>
+			<plugins>
+					<!-- 
+. ${WORKSPACE}/sources/publish/rsync.sh -s ${WORKSPACE}/sources/aggregate/coretests-site/target/fullSite -t mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}
+. ${WORKSPACE}/sources/publish/rsync.sh -s ${WORKSPACE}/sources/aggregate/coretests-site/target/fullSite/all/repo -t mars/snapshots/updates/coretests/${jbosstools_site_stream}
+					-->
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>exec-maven-plugin</artifactId>
+					<version>1.3.2</version>
+					<executions>
+						<execution>
+							<id>deploy-fullSite-to-snapshot</id>
+							<goals>
+								<goal>exec</goal>
+							</goals>
+							<phase>deploy</phase>
+							<configuration>
+								<executable>${project.build.directory}/releng-scripts/publish/rsync.sh</executable>
```

Comment:
Yes, good idea. Actually, we could set it in the default configuration of the exec-maven-plugin in parent pom, so that individual executions would inherit it.